README.md

```diff
@@ -4,29 +4,55 @@
 [![Build Status](https://travis-ci.org/jupyter/nbgrader.svg)](https://travis-ci.org/jupyter/nbgrader)
 [![codecov.io](http://codecov.io/github/jupyter/nbgrader/coverage.svg?branch=master)](http://codecov.io/github/jupyter/nbgrader?branch=master)
 
-A system for assigning and grading notebooks.
+A system for assigning and grading Jupyter notebooks.
 
 [Documentation can be found on Read the Docs.](http://nbgrader.readthedocs.org)
 
+## Visual highlights of nbgrader
+
+### Command line tools
+Command lines tools offer an efficient way for the user to generate, assign,
+release, collect, and grade notebooks.
+![nbgrader system and command line tools](docs/source/user_guide/images/read_only.png "nbgrader system and command line tools")
+
+### Toolbar extension for Jupyter notebooks
+The nbgrader toolbar extension for Jupyter notebooks guides the user to do
+assignment and grading tasks within the familiar Jupyter notebook interface.
+![nbgrader toolbar extension for Jupyter notebooks](docs/source/user_guide/images/assignment_toolbar.png "nbgrader toolbar extension for Jupyter notebooks")
+
+![nbgrader assignment list](docs/source/user_guide/images/assignment_list_downloaded_expanded.png "nbgrader assignment list")
+
+
 ## Installation
 
-If you want to develop on nbgrader, please follow the [development installation instructions](CONTRIBUTING.md#development-installation).
-Otherwise, you can install the current version of nbgrader with:
+### The nbgrader system and command line tools
+You may install the current version of nbgrader which includes the grading
+system and command line tools:
 
     pip install nbgrader
 
-You can then install and activate the nbgrader assignment toolbar extension with:
+### The nbgrader toolbar extension for Jupyter notebook
+You may then install and activate the nbgrader assignment toolbar extension for
+Jupyter notebook with:
 
     nbgrader extension install
     nbgrader extension activate
```

Comment:
This is something I just realized from dicussing with someone else, but the "activate" command needs to be run for each user that wants to use the extension. Since you're editing this file anyway would you mind adding a sentence about that?